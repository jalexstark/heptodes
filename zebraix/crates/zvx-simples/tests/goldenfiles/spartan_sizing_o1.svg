<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<defs>
<g>
<g id="glyph-0-0">
<path d="M 0.375 -2.140625 L 0.375 -2.875 L 2.640625 -2.875 L 2.640625 -2.140625 Z M 0.375 -2.140625 "/>
</g>
<g id="glyph-0-1">
<path d="M 5.15625 -1.515625 L 4.1875 -1.515625 L 4.1875 0 L 3.390625 0 L 3.390625 -1.515625 L 0.203125 -1.515625 L 0.203125 -2.21875 L 3.328125 -6.703125 L 4.1875 -6.703125 L 4.1875 -2.25 L 5.15625 -2.25 Z M 3.390625 -4.359375 C 3.390625 -4.679688 3.394531 -4.953125 3.40625 -5.171875 C 3.414062 -5.398438 3.425781 -5.617188 3.4375 -5.828125 L 3.40625 -5.828125 C 3.351562 -5.710938 3.289062 -5.582031 3.21875 -5.4375 C 3.144531 -5.300781 3.070312 -5.191406 3 -5.109375 L 1 -2.25 L 3.390625 -2.25 Z M 3.390625 -4.359375 "/>
</g>
<g id="glyph-0-2">
<path d="M 0.671875 -0.5 C 0.671875 -0.726562 0.726562 -0.890625 0.84375 -0.984375 C 0.957031 -1.078125 1.09375 -1.125 1.25 -1.125 C 1.40625 -1.125 1.539062 -1.078125 1.65625 -0.984375 C 1.769531 -0.890625 1.828125 -0.726562 1.828125 -0.5 C 1.828125 -0.28125 1.769531 -0.117188 1.65625 -0.015625 C 1.539062 0.078125 1.40625 0.125 1.25 0.125 C 1.09375 0.125 0.957031 0.078125 0.84375 -0.015625 C 0.726562 -0.117188 0.671875 -0.28125 0.671875 -0.5 Z M 0.671875 -0.5 "/>
</g>
<g id="glyph-0-3">
<path d="M 4.890625 -3.34375 C 4.890625 -2.625 4.8125 -2.007812 4.65625 -1.5 C 4.507812 -0.988281 4.273438 -0.59375 3.953125 -0.3125 C 3.628906 -0.0390625 3.195312 0.09375 2.65625 0.09375 C 1.90625 0.09375 1.347656 -0.207031 0.984375 -0.8125 C 0.628906 -1.425781 0.453125 -2.269531 0.453125 -3.34375 C 0.453125 -4.0625 0.523438 -4.675781 0.671875 -5.1875 C 0.816406 -5.707031 1.050781 -6.101562 1.375 -6.375 C 1.707031 -6.644531 2.132812 -6.78125 2.65625 -6.78125 C 3.40625 -6.78125 3.960938 -6.476562 4.328125 -5.875 C 4.703125 -5.269531 4.890625 -4.425781 4.890625 -3.34375 Z M 1.28125 -3.34375 C 1.28125 -2.425781 1.382812 -1.738281 1.59375 -1.28125 C 1.800781 -0.832031 2.15625 -0.609375 2.65625 -0.609375 C 3.15625 -0.609375 3.515625 -0.832031 3.734375 -1.28125 C 3.953125 -1.738281 4.0625 -2.425781 4.0625 -3.34375 C 4.0625 -4.25 3.953125 -4.929688 3.734375 -5.390625 C 3.515625 -5.847656 3.15625 -6.078125 2.65625 -6.078125 C 2.15625 -6.078125 1.800781 -5.847656 1.59375 -5.390625 C 1.382812 -4.929688 1.28125 -4.25 1.28125 -3.34375 Z M 1.28125 -3.34375 "/>
</g>
<g id="glyph-0-4">
<path d="M 3.3125 0 L 2.515625 0 L 2.515625 -4.65625 C 2.515625 -4.925781 2.515625 -5.140625 2.515625 -5.296875 C 2.523438 -5.460938 2.535156 -5.632812 2.546875 -5.8125 C 2.453125 -5.707031 2.363281 -5.617188 2.28125 -5.546875 C 2.195312 -5.484375 2.09375 -5.398438 1.96875 -5.296875 L 1.265625 -4.71875 L 0.828125 -5.265625 L 2.640625 -6.671875 L 3.3125 -6.671875 Z M 3.3125 0 "/>
</g>
<g id="glyph-0-5">
<path d="M 4.859375 0 L 0.453125 0 L 0.453125 -0.6875 L 2.203125 -2.453125 C 2.535156 -2.785156 2.816406 -3.082031 3.046875 -3.34375 C 3.273438 -3.601562 3.445312 -3.859375 3.5625 -4.109375 C 3.6875 -4.367188 3.75 -4.644531 3.75 -4.9375 C 3.75 -5.3125 3.640625 -5.59375 3.421875 -5.78125 C 3.203125 -5.96875 2.914062 -6.0625 2.5625 -6.0625 C 2.238281 -6.0625 1.953125 -6.003906 1.703125 -5.890625 C 1.460938 -5.785156 1.21875 -5.628906 0.96875 -5.421875 L 0.515625 -5.984375 C 0.785156 -6.203125 1.085938 -6.382812 1.421875 -6.53125 C 1.765625 -6.6875 2.144531 -6.765625 2.5625 -6.765625 C 3.1875 -6.765625 3.679688 -6.609375 4.046875 -6.296875 C 4.410156 -5.984375 4.59375 -5.546875 4.59375 -4.984375 C 4.59375 -4.640625 4.519531 -4.3125 4.375 -4 C 4.226562 -3.695312 4.023438 -3.394531 3.765625 -3.09375 C 3.515625 -2.800781 3.21875 -2.488281 2.875 -2.15625 L 1.484375 -0.78125 L 1.484375 -0.75 L 4.859375 -0.75 Z M 4.859375 0 "/>
</g>
</g>
</defs>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 18.183594 286.363281 L 18.183594 13.636719 L 381.816406 13.636719 L 381.816406 286.363281 Z M 18.183594 286.363281 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 18.183594 150 L 381.816406 150 M 18.183594 204.546875 L 381.816406 204.546875 M 18.183594 259.089844 L 381.816406 259.089844 M 18.183594 95.453125 L 381.816406 95.453125 M 18.183594 40.910156 L 381.816406 40.910156 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 290.910156 286.363281 L 290.910156 13.636719 M 230.304688 286.363281 L 230.304688 13.636719 M 169.695312 286.363281 L 169.695312 13.636719 M 109.089844 286.363281 L 109.089844 13.636719 M 48.484375 286.363281 L 48.484375 13.636719 M 351.515625 286.363281 L 351.515625 13.636719 "/>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="40.484375" y="298.229492"/>
<use xlink:href="#glyph-0-1" x="43.484375" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="48.484375" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="51.484375" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="284.410156" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="289.410156" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="292.410156" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-4" x="345.015625" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="350.015625" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="353.015625" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="-2.492188" y="261.866211"/>
<use xlink:href="#glyph-0-5" x="0.507812" y="261.866211"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="261.866211"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="261.866211"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="0.507812" y="152.776367"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="152.776367"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="152.776367"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-5" x="0.507812" y="43.686523"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="43.686523"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="43.686523"/>
</g>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 168.765625 83.882812 C 168.765625 87.617188 163.167969 87.617188 163.167969 83.882812 C 163.167969 80.152344 168.765625 80.152344 168.765625 83.882812 Z M 182.285156 78.949219 C 182.285156 82.683594 176.683594 82.683594 176.683594 78.949219 C 176.683594 75.214844 182.285156 75.214844 182.285156 78.949219 Z M 201.558594 74.164062 C 201.558594 77.898438 195.957031 77.898438 195.957031 74.164062 C 195.957031 70.433594 201.558594 70.433594 201.558594 74.164062 Z M 229.421875 71.898438 C 229.421875 75.628906 223.824219 75.628906 223.824219 71.898438 C 223.824219 68.164062 229.421875 68.164062 229.421875 71.898438 Z M 267.296875 79.617188 C 267.296875 83.351562 261.695312 83.351562 261.695312 79.617188 C 261.695312 75.886719 267.296875 75.886719 267.296875 79.617188 Z M 301.957031 114.78125 C 301.957031 118.511719 296.359375 118.511719 296.359375 114.78125 C 296.359375 111.046875 301.957031 111.046875 301.957031 114.78125 Z M 285.261719 184.269531 C 285.261719 188.003906 279.660156 188.003906 279.660156 184.269531 C 279.660156 180.535156 285.261719 180.535156 285.261719 184.269531 Z M 208.800781 234.792969 C 208.800781 238.527344 203.199219 238.527344 203.199219 234.792969 C 203.199219 231.0625 208.800781 231.0625 208.800781 234.792969 Z M 144.25 239.054688 C 144.25 242.789062 138.648438 242.789062 138.648438 239.054688 C 138.648438 235.320312 144.25 235.320312 144.25 239.054688 Z M 111.566406 225.234375 C 111.566406 228.96875 105.964844 228.96875 105.964844 225.234375 C 105.964844 221.5 111.566406 221.5 111.566406 225.234375 Z M 97.21875 210.25 C 97.21875 213.984375 91.617188 213.984375 91.617188 210.25 C 91.617188 206.519531 97.21875 206.519531 97.21875 210.25 Z M 91.207031 197.914062 C 91.207031 201.644531 85.605469 201.644531 85.605469 197.914062 C 85.605469 194.179688 91.207031 194.179688 91.207031 197.914062 Z M 88.914062 188.277344 C 88.914062 192.011719 83.316406 192.011719 83.316406 188.277344 C 83.316406 184.542969 88.914062 184.542969 88.914062 188.277344 Z M 88.914062 188.277344 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(60%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 165.964844 83.882812 L 168.808594 82.734375 L 171.886719 81.558594 L 175.222656 80.359375 L 178.851562 79.152344 L 182.796875 77.941406 L 187.105469 76.75 L 191.804688 75.59375 L 196.941406 74.507812 L 202.558594 73.527344 L 208.703125 72.710938 L 215.414062 72.125 L 222.730469 71.867188 L 230.671875 72.058594 L 239.230469 72.855469 L 248.363281 74.457031 L 257.949219 77.117188 L 267.78125 81.125 L 277.511719 86.8125 L 286.613281 94.523438 L 294.367188 104.546875 L 299.878906 117.03125 L 302.160156 131.882812 L 300.308594 148.652344 L 293.746094 166.496094 L 282.460938 184.269531 L 267.082031 200.738281 L 248.796875 214.84375 L 229.050781 225.925781 L 209.230469 233.789062 L 190.414062 238.625 L 173.3125 240.871094 L 158.253906 241.070312 L 145.3125 239.75 L 134.371094 237.367188 L 125.238281 234.292969 L 117.675781 230.808594 L 111.445312 227.113281 L 106.339844 223.355469 L 102.160156 219.625 L 98.75 215.996094 L 95.972656 212.496094 L 93.714844 209.15625 L 91.886719 205.984375 L 90.40625 202.980469 L 89.214844 200.148438 L 88.261719 197.480469 L 87.507812 194.964844 L 86.917969 192.601562 L 86.460938 190.375 L 86.113281 188.277344 "/>
</svg>
