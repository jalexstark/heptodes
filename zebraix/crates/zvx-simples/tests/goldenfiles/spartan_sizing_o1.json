{
  "drawables": [
    {
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "segments": [
            {
              "Polyline": [
                [
                  -4.5,
                  -2.5
                ],
                [
                  -4.5,
                  2.5
                ],
                [
                  1.5,
                  2.5
                ],
                [
                  1.5,
                  -2.5
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -4.5,
                  0.0
                ],
                [
                  1.5,
                  0.0
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                -0.0,
                -1.0
              ],
              [
                -0.0,
                -2.0
              ],
              [
                0.0,
                1.0
              ],
              [
                0.0,
                2.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  -2.5
                ],
                [
                  0.0,
                  2.5
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                -1.0,
                -0.0
              ],
              [
                -2.0,
                -0.0
              ],
              [
                -3.0,
                -0.0
              ],
              [
                -4.0,
                -0.0
              ],
              [
                1.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Diagram",
          "anchor_choice": {
            "ThreeByThree": [
              "Center",
              "Top"
            ]
          },
          "texts": [
            {
              "content": "-4.0",
              "location": [
                -4.0,
                -2.5
              ]
            },
            {
              "content": "0.0",
              "location": [
                0.0,
                -2.5
              ]
            },
            {
              "content": "1.0",
              "location": [
                1.0,
                -2.5
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Diagram",
          "anchor_choice": {
            "ThreeByThree": [
              "Right",
              "Middle"
            ]
          },
          "texts": [
            {
              "content": "-2.0",
              "location": [
                -4.5,
                -2.0
              ]
            },
            {
              "content": "0.0",
              "location": [
                -4.5,
                0.0
              ]
            },
            {
              "content": "2.0",
              "location": [
                -4.5,
                2.0
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              -2.061559595959596,
              1.212121212121212
            ],
            [
              -1.8384928511434808,
              1.3025845312087543
            ],
            [
              -1.5204902321747493,
              1.3903008655035032
            ],
            [
              -1.0607287773128775,
              1.4318921431892142
            ],
            [
              -0.4358064516129027,
              1.2903225806451608
            ],
            [
              0.1361028765626899,
              0.6457094307561587
            ],
            [
              -0.13941151832460769,
              -0.6282722513089019
            ],
            [
              -1.4010149921790416,
              -1.5545662375165452
            ],
            [
              -2.466069387755104,
              -1.6326530612244905
            ],
            [
              -3.0053747126436794,
              -1.3793103448275865
            ],
            [
              -3.2421215616922883,
              -1.1046116174670115
            ],
            [
              -3.341290276224693,
              -0.8783992767243226
            ],
            [
              -3.3790987468671694,
              -0.7017543859649125
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Polyline": {
          "path": [
            [
              -2.061559595959596,
              1.212121212121212
            ],
            [
              -2.0146597038408243,
              1.2332044631124814
            ],
            [
              -1.9638853845111426,
              1.2547722185617691
            ],
            [
              -1.9088126259627254,
              1.2767164529098287
            ],
            [
              -1.848967318435754,
              1.2988826815642456
            ],
            [
              -1.783821605828618,
              1.321054772735678
            ],
            [
              -1.712791703509212,
              1.3429349190107533
            ],
            [
              -1.635238488220917,
              1.3641172662211312
            ],
            [
              -1.5504729237665404,
              1.3840532989469156
            ],
            [
              -1.4577695280550764,
              1.4020066551748087
            ],
            [
              -1.3563927761055798,
              1.4169946746932158
            ],
            [
              -1.2456437673540597,
              1.4277138748183174
            ],
            [
              -1.1249378549466447,
              1.4324470971242536
            ],
            [
              -0.9939283003846018,
              1.4289520454125149
            ],
            [
              -0.8526960125991272,
              1.4143356887012026
            ],
            [
              -0.7020296382504118,
              1.3849287169042768
            ],
            [
              -0.5438200705442972,
              1.3361919911473819
            ],
            [
              -0.3815812720558848,
              1.2627159851909622
            ],
            [
              -0.2210719493139831,
              1.1584152058049182
            ],
            [
              -0.07091224420897957,
              1.0170635931271796
            ],
            [
              0.05704583333333368,
              0.8333333333333327
            ],
            [
              0.14800285918715805,
              0.6044247975686983
            ],
            [
              0.185652957815019,
              0.33212688950393765
            ],
            [
              0.15509169253475127,
              0.02470502362217224
            ],
            [
              0.04683347924127851,
              -0.30242737763629274
            ],
            [
              -0.1394115183246084,
              -0.6282722513089024
            ],
            [
              -0.39313154047170173,
              -0.9301710165062362
            ],
            [
              -0.6948226133742063,
              -1.1887827190650688
            ],
            [
              -1.020635766070567,
              -1.3919768003866617
            ],
            [
              -1.3477278177003225,
              -1.5361164380193753
            ],
            [
              -1.6581506125080616,
              -1.6247582205029028
            ],
            [
              -1.9403678666658875,
              -1.6659731948733891
            ],
            [
              -2.1888017610701898,
              -1.6696300059266798
            ],
            [
              -2.402372401254577,
              -1.6454159944029596
            ],
            [
              -2.5828648903624973,
              -1.6017500982721204
            ],
            [
              -2.7335767999233074,
              -1.5453935384910373
            ],
            [
              -2.8583768959435645,
              -1.481481481481482
            ],
            [
              -2.961132995899341,
              -1.4137513134199755
            ],
            [
              -3.04540985923367,
              -1.344827200661904
            ],
            [
              -3.1143409086658647,
              -1.276489275397482
            ],
            [
              -3.1706009165902858,
              -1.2098991750687447
            ],
            [
              -3.2164312736107554,
              -1.145776683148615
            ],
            [
              -3.253688492615483,
              -1.084532415114132
            ],
            [
              -3.2838994832719526,
              -1.0263647755891623
            ],
            [
              -3.308315131109071,
              -0.9713294561779412
            ],
            [
              -3.3279583709080263,
              -0.9193885406356537
            ],
            [
              -3.3436654943176616,
              -0.8704447966419832
            ],
            [
              -3.3561207310409866,
              -0.8243653654517245
            ],
            [
              -3.365884734197401,
              -0.7809979418145409
            ],
            [
              -3.373417809881363,
              -0.7401816754755336
            ],
            [
              -3.379098746867169,
              -0.7017543859649124
            ]
          ],
          "path_choices": {
            "color": "Red"
          }
        }
      }
    }
  ]
}
